### PR TITLE
add "An Empirical Investigation into Learning Bug-Fixing Patches in the Wild via Neural Machine Translation"

### PR DESCRIPTION
**Pull request by monperrus**
…

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -162,6 +162,7 @@ Code Vectors: Understanding Programs Through Embedded Abstracted Symbolic Traces
 
 #### Program Repair and Bug Detection
 
+* <img src="badges/6-pages-gray.svg" alt="6-pages" align="top"> [An Empirical Investigation into Learning Bug-Fixing Patches in the Wild via Neural Machine Translation](http://www.cs.wm.edu/~denys/pubs/ASE%2718-Learning-Bug-Fixes-NMT.pdf) - M. Tufano et al., ASE 2018
```
Comment (EgorBu) [inline README.md:165]: please add dot to the end of the line to keep style consistent.

Comment (EgorBu) [reply README.md:165]:
```
Michele Tufano
College of William and Mary
Williamsburg, VA, USA
Cody Watson
College of William and Mary
Williamsburg, VA, USA
Gabriele Bavota
Università della Svizzera italiana (USI)
Lugano, Switzerland
Massimiliano Di Penta
University of Sannio
Benevento, Italy
Martin White
College of William and Mary
Williamsburg, VA, USA
Denys Poshyvanyk
College of William and Mary
Williamsburg, VA, USA
```
there are six authors of this paper - they should be mentioned.

**Review by EgorBu — CHANGES_REQUESTED** [review]
Thanks for the article! Please signoff and update according to feedback

**Comment by EgorBu** [comment]
The article was added in https://github.com/src-d/awesome-machine-learning-on-source-code/pull/111

**Comment by monperrus** [comment]
Thanks. --Martin

**Pull request closed without merging**